lms/static/sass/xblocks/problem_builder.scss

```diff
@@ -392,6 +392,13 @@ body.new-theme {
                 box-shadow: 0px 0px 35px rgba(0, 0, 0, 0.3);
                 padding: 15px 30px 15px 15px !important;
                 top: 45px;
+                position: absolute;
+
+                .fa-times-circle {
```

Comment:
@tanveer65 The will add double icons for the feedbacks that are working fine. Please improve the selector and only apply this icon to ".new-theme [data-block-type="pb-mrq"] .questionnaire .feedback:after"